metadat/no-intro/Nintendo - Nintendo 64.dat

```diff
@@ -210,15 +210,15 @@ game (
 )
 
 game (
-	name "AeroGauge (Japan) (Rev 1)"
-	description "AeroGauge (Japan) (Rev 1)"
-	rom ( name "AeroGauge (Japan) (Rev 1).z64" size 8388608 crc F322B641 md5 0620C2D134A0430F4AFA208FFEDA67B8 sha1 F7C2D4B0E77CC02DEAFE2E5FD95152B7E8BF86CF )
+	name "AeroGauge (Japan) (Rev A)"
```

Comment:
AeroGauge uses Rev 1 in No-Intro
https://datomatic.no-intro.org/index.php?page=show_record&s=24&n=0025